app/models/requests/requestable.rb

```diff
@@ -270,6 +269,16 @@ def available?
       (always_requestable? && !held_at_marquand_library?) || item.available?
     end
 
+    def cul_avery?
+      return false unless item?
+      return false if item[:collection_code].blank?
```

Comment:
This could be written as one line and be simpler to understand in my opinion.
`item[:collection_code].present? && item[:collection_code] == 'AR'`

This code take advantage of the early exit for false.  Also, I have a hard time processing so many guard statements.  

So the entire routine would be
```
    def cul_avery?
      return false unless item?
      item[:collection_code].present? && item[:collection_code] == 'AR'
   end
```

Reply:
Thanks for catching that. I updated the code and re-submitted.